### PR TITLE
Backend/invites

### PR DESCRIPTION
**Pull request by Quat3rnion**
- Adds implementation to convert `Guild` to `InviteGuild`
- Adds GetInviteSchema
- Replaces primitives with distinct types on `InviteGuild`

### CODE REVIEW

**Comment by Quat3rnion** [comment]
forgot to include this in the last pr, oops

**Comment by bitfl0wer** [comment]
Still recovering from my sickness + taking a few days of break time after merging that big pr yesterday, so I might not review this in time, unless it is urgent. Is that okay?

File: src/types/schema/invites.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq, PartialOrd, Eq, Ord)]
+/// Query parameters for routes that list invites.
+/// # Reference:
+/// Read: <https://discord-userdoccers.vercel.app/resources/invite#invite-structure>
```
Comment (kozabrada123) [inline src/types/schema/invites.rs:6]: Minor change to make the docs prettier and afaik this only for the one `Get Invite` route

```suggestion
/// Query parameters for the `Get Invite` route.
///
/// # Reference:
/// Read: <https://docs.discord.sex/resources/invite#query-string-params>
```

**Review by kozabrada123 — APPROVED** [review]
Looks good, I have a minor suggestion for the docs of the schema though :D

**Comment by kozabrada123** [comment]
> Still recovering from my sickness + taking a few days of break time after merging that big pr yesterday, so I might not review this in time, unless it is urgent. Is that okay?

Dw, take all the time you need

**Pull request merged**